### PR TITLE
A couple of visual fixes for the dashboard

### PR DESCRIPTION
**Pull request by corymcdonald**
#### Before
image

#### After

image

#### Features

* Fixed problem where only one chart was being shown
* Fixed issue where referral selection did not work in mobile due to no theme in brave-ui
* Fixed nil channel details
* Fixed background on the partners dashbaord

#### How To Test

1. Pull down, make yourself a partner
2. Go to the dashboard and check things out
3. Add a youtube channel and then active referral promo
4. Run `rails database_updates:mock_data:populate_promo_stats` to generate some referral stats you can view on the homepage

### CODE REVIEW

File: app/javascript/packs/referral_charts.jsx [hunk 1]
```diff
@@ -1,209 +1,93 @@
...
+    this.bindFunctions();
   }
 
-  bindFunctions () {
-    this.viewReferralCodeStats = this.viewReferralCodeStats.bind( this )
+  bindFunctions() {
+    this.viewReferralCodeStats = this.viewReferralCodeStats.bind(this);
   }
 
-  createLabels ( startingDate ) {
```
Comment (corymcdonald) [inline app/javascript/packs/referral_charts.jsx]: All these methods were moved to a subcomponent. It was a little bit of a cleaner React-esque way to handle these updates 🎉

File: app/javascript/packs/chart/Chart.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            return numericalValue >= 1 ? numericalValue.toLocaleString() : "";
+          }
+          return value;
+        }
+      }
+    });
+  }
+
+  componentDidUpdate(prevProps) {
+    console.log(this.props.title);
```
Comment (yachtcaptain23) [inline app/javascript/packs/chart/Chart.js]: Debug statement.

Comment (corymcdonald) [reply app/javascript/packs/chart/Chart.js]: https://github.com/brave-intl/publishers/pull/1807/commits/a783322a7e8f93db149277fe43bec6f824cefae8

File: app/javascript/packs/referral_charts.jsx [hunk 3]
```diff
@@ -1,209 +1,93 @@
...
-    const node = this.selectMenuRef.current
+  async viewReferralCodeStats() {
+    const node = this.selectMenuRef.current;
     var url = routes.publishers.promo_registrations.show.path.replace(
       "{id}",
-      document.getElementById( "publisher_id" ).value
-    )
-    url = url.replace( "{referral_code}", node.state.value )
-    const result = await fetch( url, {
+      document.getElementById("publisher_id").value
```
Comment (dlipeles) [inline app/javascript/packs/referral_charts.jsx]: Any way we could pass this in as a prop?

Comment (corymcdonald) [reply app/javascript/packs/referral_charts.jsx]: Done! https://github.com/brave-intl/publishers/pull/1807/commits/ce6476cb973b1e5a281a9233593a58803e588ee9

**Pull request merged**